src/main/app/intercom.ts

```diff
@@ -114,7 +113,7 @@ export function handleWelcomeScreenModal() {
     }
 }
 
-export function handleMentionNotification(event: IpcMainEvent, title: string, body: string, channel: {id: string}, teamId: string, url: string, silent: boolean, data: MentionData) {
+export function handleMentionNotification(event: IpcMainEvent, title: string, body: string, channel: {id: string}, teamId: string, url: string, silent: boolean, data: {soundName: string}) {
```

Comment:
Some types, like `MentionData` and some of the calls types weren't as necessary since we have more control over what is sent up and down through the API, so I've removed most of the extra ones.